commands/delete/command.go

```diff
@@ -12,11 +13,12 @@ var (
 	Command = &cobra.Command{
 		Use:   "delete",
 		Short: "Delete things",
-		Long:  `Lets you delete a cluster or node pool`,
+		Long:  `Lets you delete a cluster, a node pool or an API endpoint`,
```

Comment:
```suggestion
		Long:  `Lets you delete a cluster, a node pool, or an API endpoint`,
```

Up in these woods we use the oxford comma :P

🇬🇧